packages/frontend/src/config/configFromEnvironment.js

```diff
@@ -72,4 +74,7 @@ module.exports = {
     WHITELISTED_CONTRACTS: parseCommaSeperatedStringAsArrayFromShell(
         process.env.TOKEN_CONTRACTS
     ),
+    NEAR_TOKEN_ID: process.env.NEAR_TOKEN_ID || 'wrap.testnet',
```

Comment:
1. Because `wrap.testnet` only applies to the testnet environment; we shouldn't have a default like this here -- instead, you will need to add an entry for `NEAR_TOKEN_ID` to the `config/environmentDefaults/testnet` file
2. To ensure that we display the correct value on mainnet, you will also have to add the appropriate contract id to the `config/environmentDefaults/mainnet` and `config/environmentDefaults/mainnet_STAGING` files